docs/store/datastore.md

```diff
@@ -244,31 +223,52 @@ The equivalent to this parameter in environment authentication is "GCP_CREDENTIA
 
 The code prioritizes `gcp_credentials` over `credentials_path`.
 
+### GCS credentials and parameters
+* `GOOGLE_APPLICATION_CREDENTIALS` &mdash; path to the application credentials to use (in the form of a JSON file). This can
+be used if this file is located in a location on shared storage, accessible to pods executing MLRun jobs.
+* `GCP_CREDENTIALS` &mdash; when the credentials file cannot be mounted to the pod, this secret or environment variable
+may contain the contents of this file. If configured in the function pod, MLRun dumps its contents to a temporary file 
+and points `GOOGLE_APPLICATION_CREDENTIALS` at it. An exception is `BigQuerySource`, which passes `GCP_CREDENTIALS`'s
+contents directly to the query engine.
 
-### Kafka data store profile
+## HDFS data store profile
```

Comment:
section ## HDFS data store
Here you can mention that we don't support HDFS with env-vars, only with datastore profile
subsection ### HDFS datastore profile